example/spec/parallel/changeGranuleCollections/ChangeGranuleCollectionWorkflowSpec.js

```diff
@@ -73,41 +86,40 @@ describe('The ChangeGranuleCollections workflow', () => {
       targetCollection.name,
       targetCollection.version
     );
-    provider = { id: `s3_provider${testSuffix}` };
+    provider = { id: 's3_provider' };
 
     // populate collections, providers and test data
-    await Promise.all([
-      uploadTestDataToBucket(config.bucket, s3data, testDataFolder),
-      addCollections(
-        stackName,
-        config.bucket,
-        collectionsDir,
-        testSuffix,
-        testId
-      ),
-      addCollections(
-        stackName,
-        config.bucket,
-        targetCollectionsDir,
-        testSuffix,
-        testId
-      ),
-      addProviders(
-        stackName,
-        config.bucket,
-        providersDir,
-        config.bucket,
-        testSuffix
-      ),
-    ]);
+    try {
+      await Promise.all([
+        uploadTestDataToBucket(config.bucket, s3data, testDataFolder),
+        addCollections(
+          stackName,
+          config.bucket,
+          collectionsDir
+        ),
+        addCollections(
+          stackName,
+          config.bucket,
+          targetCollectionsDir
+        ),
+        addProviders(
+          stackName,
+          config.bucket,
+          providersDir,
+          config.bucket
+        ),
+      ]);
+    } catch (error) {
+      console.log('error thrown in data setup', error);
+    }
 
     const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
     // update test data filepaths
     inputPayload = await setupTestGranuleForIngest(
       config.bucket,
       JSON.stringify({ ...JSON.parse(inputPayloadJson), pdr: undefined }),
       granuleRegex,
-      testSuffix,
+      '',
```

Comment:
Note: Reading through this method.... this should probably be fine....although based on the table structure of the PDRs table, we are probably updating the *same record* each time in the PDRs table if it's not cleaned up.   I can't see any reason that causes test instability,